packages/webdriver/src/command.ts

```diff
@@ -101,7 +101,7 @@ export default function (
 
             this.emit('result', { method, endpoint, body, result })
 
-            if (command === 'deleteSession') {
+            if (command === 'deleteSession' && process.env.WDIO_WORKER_ID) {
```

Comment:
```suggestion
            if (command === 'deleteSession' && !process.env.WDIO_WORKER_ID) {
```

Reply:
Updated as suggested, and also updated unit tests